README.md

```diff
@@ -1 +1,157 @@
-# budget-tracker
\ No newline at end of file
+![](https://img.shields.io/badge/thecodechaser-blueviolet)
+
+# Recipe-App
```

Comment:
- [OPTIONAL] I think the name of your app should be something like `Budget tracker App`, please kindly update your project name by giving it a more meaningful name that is related to the project, as this will make your `readme file` more informative and professional.

Reply:
@t-yanick Thank you for pointing it our, I changed it now